libwhisper/BloomFilter.h

```diff
@@ -83,12 +86,51 @@ void TopicBloomFilterBase<N>::removeRaw(FixedHash<N> const& _h)
 
 template <unsigned N>
 bool TopicBloomFilterBase<N>::isBitSet(FixedHash<N> const& _h, unsigned _index)
-{	
+{
 	unsigned iByte = _index / 8;
-	unsigned iBit = _index % 8;
+	unsigned iBit = _index & 0x7;
```

Comment:
Any reason to change this to an "or" operation? I know it's the same, but "% 8" is clear for me as the previous line is "/ 8".